services/motion/builtin/builtin.go

```diff
@@ -350,6 +356,27 @@ func (ms *builtIn) GetPose(
 	)
 }
 
+func (ms *builtIn) StopPlan(
+	ctx context.Context,
+	req motion.StopPlanReq,
+) error {
+	return errUnimplemented
+}
+
+func (ms *builtIn) ListPlanStatuses(
+	ctx context.Context,
+	req motion.ListPlanStatusesReq,
+) ([]motion.PlanStatusWithID, error) {
+	return nil, errUnimplemented
+}
```

Comment:
I decided to give all the new interface methods their own request structs for consistency. The zero values of the struct members are also the (sane) defaults, so using structs reduces how much code SDK users need to write.